.github/workflows/issue-label-assign.yml

```diff
@@ -0,0 +1,27 @@
+name: "Set Issue Label and Assignee"
```

Comment:
If you're planning on introducing mergify then that might be a better place to do the labeling and assigning of issues in order to reduce the number of GitHub actions that you need to run.  See https://github.com/geeksforsocialchange/meeting-mood/blob/main/.github/mergify.yml#L82

Reply:
Dependabot became a total nightmare for us on this repo and we just finished tidying it up so I don't think we're quite ready to re-add it yet which seems the main benefit of Mergify. Was planning on looking at it again when we relaunch publically in Feb/March time. Unless I'm missing some of the nice things Mergify can do?

Reply:
Mergify can also apply labels based on content of PRs, and in fact that's how my setup handles major version number changes raised by dependabot. You don't need dependabot to run mergify, and you don't need to actually merge anything with mergify.

The way you have it here will work just fine, but you'll want to refactor this when you do start using Dependabot and Mergify together so that your automatic labels are being applied in one place.

Reply:
Ok got it. Maybe I'll take another look. Didn't have much luck turning up good Rails starter guides and kinda gave up! All the ones I could find were way too in depth or way too minimal. Thanks.

Reply:
You might not need it to be rails-specific to get as far as creating releases. You've already got tests, you can copy most of the remaining config from imok and meeting-mood.

You could then continue deploying to staging from main, you just need to figure out how to deploy a release to prod. And that doesn't sound too difficult if it's just a git tag.

But, again, what you have here should work if you want to leave the mergify and dependabot configuration until later.